frontend/www/js_templates/api/circuit.js

```diff
@@ -10,6 +10,7 @@ async function deleteCircuit(workgroupID, circuitID, end=-1) {
     if ('error_text' in data) throw(data.error_text);
     return data.results;
   } catch(error) {
+    alert('An error occured while deleting a circuit:\n ' + error, ()=> window.location="?action=welcome", {error: true});
```

Comment:
Alerts with a redirect won't work here. Both the welcome and modify_l2vpn pages use this code.